src/test/resources/application-connected-test.yml

```diff
@@ -1,3 +1,13 @@
 workspace:
   # workspace-dev folder id to match dev service account used in testing.
   google.folder-id: 7275817190
+  spend:
+    spend-profiles:
+      -
+        # TODO(PF-186): Create a meaningful spend profile id with a resource in Sam.
+        id: foo
+        # The billing account workspace-dev has access to.
+        billing-account-id: 01A82E-CA8A14-367457
+      -
+        # A special spend profile with no billing account associated with it.
+        id: no-billing-account
```

Comment:
nothing is reading this account atm, do we need it?

Reply:
This is being used in the [WorkspaceServiceTest for handling a spend profile without a billing account](https://github.com/DataBiosphere/terra-workspace-manager/blob/8883fabc7226b8b43f59f8306b16bff967d2a59f/src/test/java/bio/terra/workspace/service/workspace/WorkspaceServiceTest.java#L287). (The SpendProfileService is being created with this id so that it's available. If we didn't have this spend profile in the config, we would not be able to get a spend profile from the service without a billing account.) Currently, this special id is hardcoded in the spend connected test utils. I agree the connection isn't obvious. I'm open to other alternatives.

Reply:
Ah good point - I saw it there but didn't follow the check all the way into the SpendProfileService.